easybuild/easyconfigs/p/PySVN/PySVN-1.9.10-GCCcore-8.2.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CmdCp'
+
+name = 'PySVN'
+version = '1.9.10'
+versionsuffix = '-python'
+
+homepage = 'https://pysvn.sourceforge.io/'
+description = """The PySVN project's goal is to enable Tools to be written in Python that use Subversion source control system."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['https://netcologne.dl.sourceforge.net/project/pysvn/pysvn/V%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['bc66c05bd0c2d9ea0bac85bed248945df2818ea796ece8f9ad6d97b01a39ab8e']
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+dependencies = [
+    ('Subversion', '1.12.0'),
+    ('PyCXX', '7.1.3', '-python')
+]
+
+files_to_copy = [
+	(["Source/pysvn/__init__.py", "Source/pysvn/_pysvn*.so"], "python-libdir/site-packages/pysvn")
+]
+
+cmds_map = [('.*', 'cd Source/ && python setup.py configure --pycxx-src-dir=$EBROOTPYCXX/share/python%(pyshortver)s/CXX/ --svn-inc-dir=$EBROOTSUBVERSION/include/subversion-1/ --svn-lib-dir=$EBROOTSUBVERSION/lib/ --svn-bin-dir=$EBROOTSUBVERSION/bin --apr-inc-dir=$EBROOTAPR/include/apr-1/ --apu-inc-dir=$EBROOTAPRMINUTIL/include/apr-1/ --apr-lib-dir=$EBROOTAPR/lib/ && make')]
```

Comment:
@casio888 I get this error during the first iteration (using Python 3.7.2)

```
Error: Cannot find PyCXX include CXX/Version.hxx - use --pycxx-dir
```

I think you should specify a different location to `--pycxx-dir`, probably `include/python3.7m` when building with Python 3, and `include/python2.7` when building with Python 2, so maybe try this:

```
python setup.py configure  --pycxx-src-dir=$EBROOTPYCXX/include/pyython%(pyshortver)s*/
```

The trailing `*` is important, because of the `m` that is needed when using Python 3.7.2...